plugins/single-instance/examples/vanilla/src-tauri/Cargo.toml

```diff
@@ -1,5 +1,7 @@
+[workspace]
+
 [package]
-name = "single-instance-example"
+name = "app"
```

Comment:
Looks like a change in different plugin.

Reply:
true, though the name really doesn't matter much

Reply:
```suggestion
name = "single-instance-example"
```